model-optimizer/extensions/front/onnx/flattenONNX_to_reshape.py

```diff
@@ -51,28 +51,30 @@ def replace_sub_graph(self, graph: Graph, match: dict):
         assert node.has_valid('axis'), 'Flatten {} should have `axis` attribute extracted, but it\'s not'.format(name)
         axis = node.axis
 
+        reshape_node = Reshape(graph, {'name': node.id + '/Reshape'}).create_node()
+
         if axis == 0:
-            dim = Const(graph, {'value': int64_array([1, -1])}).create_node()
+            dim = Const(graph, {'value': int64_array([1, -1]), 'name': reshape_node.name + '/shape'}).create_node()
         elif axis == 1:
-            dim = Const(graph, {'value': int64_array([0, -1])}).create_node()
+            dim = Const(graph, {'value': int64_array([0, -1]), 'name': reshape_node.name + '/shape'}).create_node()
         else:
             shape = Shape(graph, {'name': name + '/input_shape'}).create_node()
 
             idxs = list(range(axis)) if axis > 0 else list(range(axis, 0))
 
             axis_shape_portion = node_to_get_shape_value_of_indices(shape, idxs)
             first_dims = create_op_node_with_second_input(graph, ReduceProd, int64_array([0]),
-                                                          {'keep_dims': True})
-            second_dims = Const(graph, {'value': int64_array([-1])}).create_node()
+                                                          {'name': name + '/first_dims', 'keep_dims': True})
```

Comment:
there is no variable called `name` in this scope. Looks like this code is not covered in the pre-commit.

Reply:
line 49: `name = node.soft_get('name', node.id)`